server.py

```diff
@@ -224,12 +249,37 @@ def getStickers():
             return json.dumps({'status': '403', 'error': 'Not authenticated or cookies disabled.'}), 405
     if (request.args.get('west') != '' and request.args.get('east') != '' and request.args.get('north') != '' and request.args.get('south') != ''):
         # Get all the stickers within the bounding box
-        with psycopg2.connect() as con:
+        with psycopg2.connect(host=POSTGRES_HOST, dbname=POSTGRES_DBNAME, user=POSTGRES_USER, password=POSTGRES_PASS, port=POSTGRES_PORT) as con:
             # create cursor
             cursor = con.cursor()
             # find results
-            rows = cursor.execute("SELECT * FROM stickers WHERE stickerLat BETWEEN %s AND %s AND stickerLon BETWEEN %s AND %s AND verified='1'", (request.args.get('south'), request.args.get('north'), request.args.get('west'), request.args.get('east'))).fetchall()

```

Comment:
verified a feature that existed when I started working on the project. It means that stickers must first be approved before they appear on the map. I (temporarily) removed that, because it made it easier to test and the admin page isn't working yet so 'approving' isn't yet possible

Reply:
Even though this is temporary, I understand that the approval system must be added later anyway, so for now it is ok to just removed these things if they are in the way